src/data/matrix/basic.lean

```diff
@@ -303,58 +303,62 @@ variable [fintype m]
 def dot_product [has_mul α] [add_comm_monoid α] (v w : m → α) : α :=
 ∑ i, v i * w i
 
+/- the precedence of ` ⬝ᵥ ` for `matrix.dot_product` is set as to come
+   immediately after ` • ` for `has_scalar.smul` -/
```

Comment:
```suggestion
/- The precedence of 72 comes immediately after ` • ` for `has_scalar.smul`, so that
`r₁ • a ⬝ᵥ r₂ • b` is parsed as `(r₁ • a) ⬝ᵥ (r₂ • b)` -/
```